### PR TITLE
bump to core 1.0

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

**Review by Noarkhh — COMMENTED** [review]
I believe we can wait until I merge plugins to use Precompiled Dependency Provider

**Comment by mat-hek** [comment]
@Noarkhh the dependency provider stuff is all done, right?

File: README.md
Comment (Noarkhh) [inline README.md]: why `github:`?

**Review by Noarkhh — APPROVED** [review]

**Pull request merged**
